docs/porting/target/cmake.md

```diff
@@ -0,0 +1,124 @@
+# CMake target porting
+
+
+Important variables provided by for Mbed target:
+
+- `MBED_TARGET_LABELS`: a list of labels added to a given Mbed target coming from the Mbed target default configuration file `targets.json`.
+- `MBED_TARGET_DEFINITIONS`: a list of macro definitions added to a given Mbed target coming from the Mbed target default configuration file `targets.json`.
+
+- `MBED_CONFIG_DEFINITIONS`: a list of macro definitions added to a given Mbed target coming from various Mbed library configuration files.
+- `MBED_TARGET_SUPPORTED_C_LIBS`: a list of C library types supported by a given Mbed target coming from the Mbed target default configuration file `targets.json`.
+- `MBED_TARGET_SUPPORTED_APPLICATION_PROFILES` a list of application profiles supported by a given Mbed target coming from the Mbed target default configuration file `targets.json`.
+
+## Mbed targets CMake input file structure
+
+### Vendor selection
+
+Vendor selection is handled in [`targets/CMakeLists.txt`](LINK) using a conditional statement to verify that one of the labels used by a given Mbed target match a given partner directory. The matching directory is added to directories to build.
```

Comment:
The hyperlink is not working. The URL is needed.

Reply:
wait a second, just pushed another commit fixing it